gatsby-config.js

```diff
@@ -43,6 +43,14 @@ module.exports = {
         path: `${__dirname}/content/homepage`,
       },
     },
+    {
+      resolve: 'gatsby-source-filesystem',
+      options: {
+        name: 'community',
+        path: `${__dirname}/content/community`,
+        include: ['**/*.md'], // ignore files starting with a dot
```

Comment:
Include is not a valid option https://www.gatsbyjs.com/plugins/gatsby-source-filesystem/#options

Reply:
Fixed!